roofit/roofitcore/test/testRooFuncWrapper.cxx

```diff
@@ -0,0 +1,52 @@
+#include <RooRealVar.h>
+#include <RooWorkspace.h>
+#include <RooRealProxy.h>
+#include <RooGaussian.h>
+#include <RooFuncWrapper.h>
+#include <RooDataSet.h>
+#include <RooAbsPdf.h>
+#include <TROOT.h>
+#include <TSystem.h>
+#include <RooFitResult.h>
+#include <RooListProxy.h>
+#include <TMath.h>
+
+#include "gtest/gtest.h"
+
+class Interface : public ::testing::Test {};
+
+TEST(Interface, createFuncWrapper)
```

Comment:
```suggestion
TEST(RooFuncWrapper, GaussianNormalized)
```
A less general name please. Look at the other tests in RooFitCore to see what the name convention is: first we have the name of the class that is tested, and then what is tested, both capitalized. And `createFuncWrapper` is probably not such a good name because all of the tests that will be added in this file will create a function wrapper instance.